jormungandr/src/rest/v0/mod.rs

```diff
@@ -42,11 +42,24 @@ pub fn filter(
         root.and(get.or(get_next)).boxed()
     };
 
-    let fragment = warp::path!("fragment" / "logs")
-        .and(warp::get())
-        .and(with_context.clone())
-        .and_then(handlers::get_message_logs)
-        .boxed();
+    let fragment = {
+        let root = warp::path!("fragment" / ..).boxed();
+
+        let logs = warp::path!("logs")
+            .and(warp::get())
+            .and(with_context.clone())
+            .and_then(handlers::get_message_logs)
+            .boxed();
+
+        let statuses = warp::path!("statuses")
+            .and(warp::post())
```

Comment:
**GET** would be better instead of **POST** method

Reply:
I agree with that

Reply:
We can do that. I made it `POST` because the HTTP standard does not recommend parsing `GET` bodies. But I agree `GET` is semantically better here.

Reply:
Good point, I've missed that part. POST is fine.

Reply:
By the way, why not use `GET` queries? In a form if `?ids=id1,id2,id3` or `?id=id1&id=id2&id=id3`.

Reply:
So they are going to be comma-separated

Reply:
You could do them both actually, **post** for json and **get** `?ids=id1,id2,id3`.
What would be the response type/fromat of that get option?

Reply:
the same